core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="dot-tab-buttons">
+    <div class="dot-tab-buttons__container" *ngFor="let item of options">
+        <div class="dot-tab" (click)="onClickOption($event)" data-testId="dot-tab-container">
+            <button
+                class="dot-tab__button"
+                [ngClass]="{
+                    'dot-tab--active': mode === item.value,
+                    'dot-tab__button--right': item.value === pageMode.PREVIEW
+                }"
+                [value]="item.value"
+                [disabled]="item.disabled"
+                data-testId="dot-tab-button-text"
+            >
+                {{ item.label }}
+            </button>
+
+            <button
+                class="dot-tab__dropdow"
+                *ngIf="item.value === pageMode.PREVIEW"
+                [ngClass]="{ 'dot-tab-dropdown--active': mode === pageMode.PREVIEW }"
+            >
+                <i class="dot-tab-dropdown__icon" [value]="'openMenu'" [class]="icon"></i>
+            </button>
```

Comment:
Why aren't these primeNG buttons?

Reply:
@zJaaal These buttons are kind of like the split button but It doesn't have the context menu, I tried using the model as null but it always shows like a weird blank menu. 

<img width="980" alt="image" src="https://github.com/dotCMS/core/assets/3438705/4c5621e4-8106-411a-aba5-f3a9cc6ead43">

Reply:
@manuelrojas If you use directive `pButton` it breaks the behavior/styles? To keep the styles consistent in the app, we should use `pButton`

```html
<button
    pButton
    class="dot-tab__dropdow"
    ....
>
   ....
</button>
```